hack/deploy_fast.sh

```diff
@@ -1,10 +1,12 @@
-#!/bin/bash
+#!/usr/bin/env bash
+
+set -e
 
 check_ocp() {
     local KUBECONFIG_PATH=$1
 
     export KUBECONFIG=$KUBECONFIG_PATH
-    
+
     if oc get nodes; then
```

Comment:
I believe this patch is failing the fast test because we set -e on this script: https://net-hw-ci-jenkins-csb-nst.apps.ocp-c1.prod.psi.redhat.com/view/IPU%20Testing/job/99_FAST_E2E_IPU_Deploy/75/console

It is possible check_ocp() runs oc get nodes which will fail, we dont' want to bail out in this case, but rather redeploy the cluster.

Reply:
right. That is now fixed.